templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/template_schema.json

```diff
@@ -16,6 +16,22 @@
           "Ubuntu 18.04",
           "Ubuntu 18.04 Data Science VM"
         ]
+      },
+      "shared_storage_access": {
+        "$id": "#/properties/shared_storage_access",
+        "type": "string",
+        "title": "Shared Storage Access",
+        "description": "true/false on whether to set-up shared storage access",
+        "enum": [
+          "true",
+          "false"
+        ]
+      },
+      "shared_storage_name": {
```

Comment:
Can we set a default? Not sure if we should be exposing this at all. But doesn't hurt.

Reply:
Default value of "vm-shared-storage" for name and "True" for shared storage access is set in porter.  The name used is the name of the file share created as part of the workspace storage deployment.

Reply:
The json schema is used to create a UI. So if is blank the UI will provide a blank form input, and send a blank value overriding porter.

Reply:
Hmm, you might be right, images choice are loading with nothing selected. So they would definitely be better with a default. 

Thinking more I'm not sure we want/need end users to be able to specify the shared storage name as leaves a lot of potential for providing invalid values that can't be validated client side, or via the API prior to accepting the request.

Reply:
I'll omit the name from the template and just have the option to add/remove the shared storage access instead

Reply:
This has now been actioned.  Happy for me to resolve this?

Reply:
Only argument I can think pro having the end user be able to enable/disable the access would be how granular the end users would like the IG control.  Currently all shared storage is read/write so data could be moved round between VMs in a workspace.

Reply:
No problem will action that now and commit the changes.